handsontable/src/tableView.js

```diff
@@ -1298,6 +1298,10 @@ class TableView {
 
       div.className = 'relative';
       span.className = 'colHeader';
+
+      setAttribute(div, ...A11Y_PRESENTATION());
+      setAttribute(span, ...A11Y_PRESENTATION());
+
```

Comment:
```suggestion

    if (this.settings.ariaTags) {
      setAttribute(div, ...A11Y_PRESENTATION());
      setAttribute(span, ...A11Y_PRESENTATION());
    }

```
I'd be consistent with other places that add tags.